Drive/rover_drive_system.hpp

```diff
@@ -99,76 +176,165 @@ class RoverDriveSystem
+  void SetDriveMode()
+  {
+    units::angle::degree_t left_wheel_angle  = -45_deg;
+    units::angle::degree_t right_wheel_angle = -135_deg;
+    units::angle::degree_t back_wheel_angle  = 90_deg;
+    SetSpinMode();
+    left_wheel_.SetPosition(left_wheel_angle);
+    right_wheel_.SetPosition(right_wheel_angle);
+    back_wheel_.SetPosition(back_wheel_angle);
+  };
 
-  /// Sets all wheels to the speed provided. SetSpeed() handles max/min speeds
-  void SetWheelSpeed(units::angular_velocity::revolutions_per_minute_t speed)
+  /// Aligns rover wheels perpendicular to their legs using their homing mark
+  void SetSpinMode()
   {
-    left_wheel_.SetSpeed(speed);
-    right_wheel_.SetSpeed(speed);
-    back_wheel_.SetSpeed(speed);
+    left_wheel_.HomeWheel();
+    right_wheel_.HomeWheel();
+    back_wheel_.HomeWheel();
   };
 
-  // Retrieves commands for drive movement from Mission Control. Returns true if
-  // successful.
-  bool GetMissionControlData()
+  /// Aligns rover wheel all in the same direction, facing towards the right
+  void SetTranslationMode()
   {
-    return true;
-  }
+    units::angle::degree_t left_wheel_angle  = 45_deg;
+    units::angle::degree_t right_wheel_angle = -45_deg;
+    units::angle::degree_t back_wheel_angle  = -180_deg;
+    SetSpinMode();
+    left_wheel_.SetPosition(left_wheel_angle);
+    right_wheel_.SetPosition(right_wheel_angle);
+    back_wheel_.SetPosition(back_wheel_angle);
+  };
 
-  /// Sends POST to Raspberry Pi endpoint with the new rover status updates
-  void UpdateMissionControlData()
+  // =======================
+  // = DRIVE MODE HANDLERS =
+  // =======================
+
+  /// Handles drive mode. Adjusts only the rear wheel of the rover
+  void HandleDriveMode(units::angular_velocity::revolutions_per_minute_t speed,
+                       units::angle::degree_t angle)
   {
-    if (GetRoverData())
-    {
-      // POST, http://localhost:3000/drive/status, JSON.stringify(rover_data_)
-    }
-    else
-    {
-      sjsu::LogError("Unable to retrieve wheel data!");
-    }
+    back_wheel_.SetPosition(angle);
+    SetWheelSpeed(speed);
   };
 
-  /// Gets the speed of each hub motor and angle of each steer motor on the
-  /// rover. Does not get data from Mission Control scanf
-  bool GetRoverData()
+  /// Handles spin mode. Adjusts only the speed (aka the spin direction)
+  void HandleSpinMode(units::angular_velocity::revolutions_per_minute_t speed)
   {
-    // Format should be parsable by the Raspberry Pi like JSON ? Ex:
-    // {"left_wheel_speed": "x_rpm", "right_wheel_speed": "y_rpm", ...}
-    // rover_data_ = "Some key:value data struct?";
-    return true;
+    SetWheelSpeed(speed);
+  };
+
+  /// Handles translation mode. Adjusts all the wheels, keeping them parallel
+  void HandleTranslationMode(
+      units::angular_velocity::revolutions_per_minute_t speed,
+      units::angle::degree_t angle)
+  {
+    left_wheel_.SetPosition(angle);
```

Comment:
```suggestion
    left_wheel_.SetSteeringAngle(angle);
```

I feel like SetSteeringAngle make this more obvious what is happening here.